src/Akeneo/Component/StorageUtils/Cursor/CursorInterface.php

```diff
@@ -9,6 +9,6 @@
  * @copyright 2015 Akeneo SAS (http://www.akeneo.com)
  * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
  */
-interface CursorInterface extends \Countable, \Iterator
```

Comment:
even if we don't use it, personally I find it useful for custom projects
but maybe I'm wrong

typically, we could use it in our commands (like `pim:product:query`)
we display "19097 products found" and just display the first 100, instead of displaying everything